TACC/Core-Styles
# hotfix: django-cms-forms unstyled in demo

## Overview

Load `django-cms-forms` custom styles in demo.

## Related

- noticed during #267

## Changes

- **removed** duplicate code
- **added** `supportStyles` so `django-cms-forms` loads its custom code
- **fixed** incomplete comment

## Testing

1. Open http://localhost:3000/components/preview/django-cms-forms.
2. Press [Toggle Field Errors] button (at bottom of page).
3. Verify errors appear just above respective help text (**not** respective above label).

## UI

| before | after |
| - | - |
| ![before](https://github.com/TACC/Core-Styles/assets/62723358/f4936bb7-f361-4936-934a-278f884f3442) | ![after](https://github.com/TACC/Core-Styles/assets/62723358/1f2d3da9-e4bd-4afb-af62-894bc48b60fe) |
